packages/discord.js/src/structures/MessageEmbed.js

```diff
@@ -88,7 +88,8 @@ class MessageEmbed {
      * @type {?number}
      */
     // Date.parse() cannot be used here because data.timestamp might be a number
-    this.timestamp = 'timestamp' in data ? new Date(data.timestamp).getTime() : null;
+    // Additionally, the nullish coalescing operator cannot be used here as we're checking for 0
```

Comment:
IMO this comment is very much unnecessary. It's pretty obvious we're *only* checking for 0 since `<Date>.getTime()` will always return a number